packages/ckeditor5-core/src/editor/editor.js

```diff
@@ -220,6 +220,31 @@ export default class Editor {
 		 */
 		this.keystrokes = new EditingKeystrokeHandler( this );
 		this.keystrokes.listenTo( this.editing.view.document );
+
+		/**
+		 * Current number of editors instances created.
+		 *
+		 * @private
+		 * @type {Number}
+		 */
+		Editor._editorCount = Editor._editorCount ? ++Editor._editorCount : 1;
+
+		/**
+		 * A unique editor's name.
+		 *
+		 * @observable
+		 * @member {String} #name
+		 */
+		this.name = this.generateEditorLabel();
```

Comment:
cc @Reinmar Are we OK with this kind of property on every editor instance? Could be helpful for some people but this will stay forever.

*   Also, what about the name of this property?

If not OK, let's move this whole label generation logic to `EditorUIView` maybe and hide it from the outside world.

Reply:
Regardless of 👆 the docs must explain more about what it is and where it is used and why.

Reply:
As for the naming of this property. If we move it to `EditorUI` it could be renamed to `editor.ui.editorName`. And we should explain it in the comments that it's related to a11y and how it's set (via a config prop?).